partner_aging/wizard/res_partner_aging_customer.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Customer Aging search/filters -->
+    <record model="ir.ui.view" id="view_customer_aging_ad_search">
+        <field name="name">customer.aging.ad.search</field>
+        <field name="model">res.partner.aging.customer</field>
+        <field name="type">search</field>
+        <field name="arch" type="xml">
+            <search string="Customer Aging Date wise">
+                <filter
+                    string="Overdue"
+                    name="overdue"
+                    help="Overdue"
+                    domain="['|', '|', '|', '|',
+                            ('days_due_01to30', '&gt;', 0),
+                            ('days_due_31to60', '&gt;', 0),
+                            ('days_due_61to90', '&gt;', 0),
+                            ('days_due_91to120', '&gt;', 0),
+                            ('days_due_121togr', '&gt;', 0)]"
+                />
+                <field name="partner_id" string="Customer" />
+                <field name="invoice_id" />
+                <field name="invoice_ref" />
+                <field name="total" sum="Total" />
+                <group expand="0" string="Group By">
+                    <filter
+                        string="Partner"
+                        name="partner_id"
+                        domain="[]"
+                        context="{'group_by': 'partner_id'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <!-- Customer Aging tree -->
+    <record model="ir.ui.view" id="view_customer_aging_ad_tree">
+        <field name="name">customer.aging.ad.tree</field>
+        <field name="model">res.partner.aging.customer</field>
+        <field name="type">tree</field>
+        <field name="arch" type="xml">
+            <tree
+                create="false"
+                delete="false"
+                decoration-danger="max_days_overdue &gt; 0 and total &gt; 0"
+            >
+                <field name="partner_id" string="Customer" />
+                <field name="salesman" />
+                <field name="date_due" string="Invoice Date" />
+                <field name="inv_date_due" string="Due Date" />
+                <field name="max_days_overdue" />
+                <field name="invoice_id" invisible="1" />
```

Comment:
invisible="1" should be replaced with column_invisible="1" in the tree view
```suggestion
                <field name="invoice_id" column_invisible="1" />
```

Reply:
Thanks @ppyczko ! Fixed.